app/views/live/show_form.html.erb

```diff
@@ -4,51 +4,53 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l govuk-!-margin-bottom-2">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
       <%= form.name %>
     </h1>
-
     <%= render FormStatusTagDescriptionComponent::View.new(status: :live) %>
 
+    <%= render MetricsSummaryComponent::View.new(metrics_data) %>
+
+    <h2 class="govuk-heading-l">Your form</h2>
```

Comment:
Does this need to be behind feature flag or are we happy for this to show even if metric feature is disabled?

Reply:
Good question! We've changed the whole heading structure below to account for this new `h2` (basically everything has been bumped down one heading level) so I think our options are (in my order of preference):

1. keep this h2 regardless of whether the feature flag is on or off (the current solution - slightly changes the design when the feature flag is off)
2. use the feature flag in each individual heading below this to determine its level (a bit complex)
3. feature flag the whole area below (simpler to feature flag but means significant duplication beetween the two versions)
4. not feature flag the headings below, so we'd have a h1 followed by h3s with the feature flag turned off (not ideal from an accessibility/HTML structure perspective as it makes the document heirarchy harder to understand)

For me option 1 made the most sense, since it's less complex to implement and the heading still makes sense without the metrics present.

Reply:
1 for the winner, i think